### DISCUSSION TITLE
Öffnen von Links als Dialog (iframe)

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#91] [General]
Um Clickjacking zu unterbinden, werden Aufruf, die aus dem webGIS als _dialog_ aufgerufen werden, "geblockt", wenn es nicht von "same-origin" kommt. Bei uns betrifft es z.B. den Aufruf des ProcessingServers. 

- [same-origin-policy](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy)

Wenn ich es richtig verstehe:

- https://wanago.io/2022/03/21/iframe-clickjacking-x-frame-options-content-security-policy/

dann muss es in der Anwendung geregelt werden über **X-Frame-Options** bzw. **Content-Security-Policy header**, welche Seiten (processingserver) in der Anwendung (webgis)  als _dialog_ aufgerufen werden können.

**Comment by gis-eni** [comment]
Die Entsprechenden HTTP Header können über die web.config ergänzt werden
